tests/Utils.ts

```diff
@@ -121,6 +121,21 @@ export const sample_data5 = (() => {
     return JSON.parse(data);
 })();
 
+export const sample_data6 = (() => {
+    const data: string = fs.readFileSync("tests/data/Block.2.sample2.json", "utf-8");
+    return JSON.parse(data);
+})();
+
+export const sample_data7 = (() => {
+    const data: string = fs.readFileSync("tests/data/Block.4.sample2.json", "utf-8");
+    return JSON.parse(data);
+})();
+
+export const sample_data8 = (() => {
+    const data: string = fs.readFileSync("tests/data/Block.5.sample2.json", "utf-8");
+    return JSON.parse(data);
+})();
+
```

Comment:
Don't use this and use BlockManager.
It will be difficult to generate a pre-image from an already prepared sample.

Reply:
@MichaelKim20 I have tried using BlockManager, In order to test ballot API's we need consistent tx_hash and utxo_key and BlockManager returns different tx_hash and different utxo_key every time

Reply:
@ahmedrns I'll research it and deliver it to you by the end of the day.